app/models/chargeback/report_options.rb

```diff
@@ -0,0 +1,102 @@
+class Chargeback
```

Comment:
@isimluk how this works ? it is something like extension of class Chargeback ?
Should it have different name ? (or same as file) or isn't better to have in module ?